### PR TITLE
Release/0.1.0

### PR DESCRIPTION
**Pull request by LukaszKwasniewski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +0,0 @@
-# integration-adaptor-111
```
Comment (petervdm) [inline README.md:1]: You shouldn't delete this one

File: release-scripts/release.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash 
+
+set -e
+
+export BUILD_TAG=latest
```
Comment (petervdm) [inline release-scripts/release.sh:5]: No need for this BUILD_TAG, just use the RELEASE_VERSION where is is used below.

File: test-scripts/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+This will build the application container locally and start it along with ActiveMQ
+The services will listen on following ports
+
+* REST service `localhost:8080`
+* ActiveMQ: `localhost:5672`
+
+## Testing
+
+## How to run unit tests:
```
Comment (petervdm) [inline test-scripts/README.md:34]: This isn't valid for testing the release. It should only mention using the test scripts in the test-scripts/{version}/tests folders. Similar to this https://hub.docker.com/repository/docker/nhsdev/nia-nhais-adaptor

File: release-scripts/release.sh (hunk 2 above)
Comment (LukaszKwasniewski) [reply release-scripts/release.sh:5]: so a local/111:0.1.0 and nhsdev/nia-111-adaptor:0.1.0?

File: test-scripts/README.md (hunk 3 above)
Comment (LukaszKwasniewski) [reply test-scripts/README.md:34]: So remove the test and integrationTest and leave only the acceptance test?

File: release-scripts/release.sh [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+docker build -t local/111:${BUILD_TAG} .
+
+docker tag local/111:${BUILD_TAG} nhsdev/nia-111-adaptor:${RELEASE_VERSION}
+
+if [ "$1" == "-y" ];
+then
+  echo "Tagging and pushing Docker image and git tag"
+  docker push nhsdev/nia-111-adaptor:${RELEASE_VERSION}
+  git tag -a ${RELEASE_VERSION} -m "Release ${RELEASE_VERSION}"
```
Comment (petervdm) [inline release-scripts/release.sh:17]: tag a `latest` version as well and push that up

Comment (LukaszKwasniewski) [reply release-scripts/release.sh:17]: so tag it bit 0.1.0 and latest?

File: test-scripts/README.md (hunk 3 above)
Comment (petervdm) [reply test-scripts/README.md:34]: These are tests used by the test partners to test the docker images running locally. Test partners won't run the unit/integration/acceptance tests, they will just run the shell scripts - like https://github.com/nhsconnect/integration-adaptor-111/pull/59/files#diff-f195099d292a84b4ecad3b94960e5313

File: README.md (hunk 1 above)
Comment (LukaszKwasniewski) [reply README.md:1]: I have moved the Readme to test-scripts, like it is in nhais. I believe we need to README.md - general one and one for release

**Pull request merged**